index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Glauber's WebDeveloper Portfolio</title>
+  <link rel="stylesheet" href="/style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+</head>
+<body>
+
+  <header>
+    <nav class="nav-bar">
+      <a href="index.html">Welcome</a></li>
+      <img src="Assets\Images\hamburguer-menu.png">
+    </nav>
+  </header>
+
+  <section class="first-section">
+  <h1 class="fist-section__title">Hey there. I'm Glauber <br>
+    I'm a software developer</h1>
+    <p class="fist-section__text"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don't hestiate to contact me.</p>
+
+  <ul class="first-section__social">
+    <li><a href="https://linkedin.com"><img src="Assets\Images\Linkedin icon.svg"></a></li>
+    <li><a href="https://twitter.com"><img src="Assets\Images\twitter-1.svg"></a></li>
+    <li><a href="https://medium.com"><img src="Assets\Images\medium.svg"></a></li>
+    <li><a href="https://angelist.com"><img src="Assets\Images\angellist-1.svg"></a></li>
+  </ul>
```

Comment:
Nice job setting up your toolbar and headline😉😉👍 but let's make it look more like the design😉 don't forget, we are[ required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/1_setup.md) to stick to the [given design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468&t=1UOK50CwR7yiVRpS-0) in this milestone😉

your page😊
<img width="299" alt="your_page" src="https://user-images.githubusercontent.com/64643995/220416493-ed7fbb9b-21e6-4552-a4ca-5fc68c7e8f7d.png">



design😊
<img width="178" alt="designmmm___" src="https://user-images.githubusercontent.com/64643995/220416119-fb93a11f-4e74-4d58-81a0-de01cc9abae7.png">

Kindly compare both images and take note of the font properties, the spacings, the positioning of the social links and make the necessary changes so that our project will look more like the design😎😉😉